json/TeemIpDiscoveryIPApplicationCollector.json

```diff
@@ -226,8 +226,8 @@
 		},
 		{
 			"attcode": "uuid",
-			"update": "0",
```

Comment:
This attribute should not be updated.

Reply:
It would mark the field as locked because of the synchro.

Reply:
It will be shown as locked in the data synchro policy page but will remain "free to be updated" in IP Application instances, like other fields location, description...

Reply:
When? When setting update to 0?

Reply:
When set to 0,  yes.

Reply:
When update is set to zero, the value of update policy is not taken into account.

Reply:
True. But the uuid field is set at creation time of the IP Application and set to r/o by the datamodel anyway.

Reply:
Correct. With my addition here, it will then also show the 🔒 with popup info on which source its synchronised with..

Reply:
The purpose of the🔒 next to an attribute in a "modify" form is to show that the attribute is actually set, and possibly updated, through a synchro. This is not the case of the UUID attribute since, as you know, it is set at the time the IP Application is created. Displaying that 🔒 would be misleading. Furthermore, the global lock displayed next to the IP Application's name, in a "detailed" view, already tells which data source it is synchronized with.

Therefore, I still think that no update of that attribute should be allowed from the synchro source, even if such update would be denied due to the r/o nature of the attribute.

Reply:
Okay, I will revert that..

Reply:
Thanks !